src/_data/events.yaml

```diff
@@ -0,0 +1,4 @@
+- type: "Webinar"
+  title: "Deploy FlowFuse on Industrial IoT with NCD.io"
+  buttonText: "Learn more"
+  link: "/webinars/2024/deploy-flowfuse-on-industrial-iot-with-ncd-io/"
```

Comment:
Maybe we should add two timestamps. Not before, that is from what point we can show the banner, and and expire. That saves a bunch of hassle each month

Reply:
Great idea ZJ, thanks! I've included an expiration date field. Since we won't be adding events or announcements before the URL is created, I believe it's sufficient for the banner to start showing when the info it's added or updated. The crucial aspect is setting when it will disappear. WDYT?